### PR TITLE
Added /version support

### PR DESCRIPTION
**Pull request by Temetz**
Ability to check API and IMG versions via API.

### CODE REVIEW

**Comment by n4sjamk-docker** [comment]
Can one of the admins verify this patch?

File: routes/version.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var middleware  = require('../middleware');
```
Comment (melonmanchan) [inline routes/version.js]: Extra space here! :full_moon_with_face:

File: routes/version.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+var Router      = require('express').Router();
+var request     = require('superagent');
+
+const IMG_URL = process.env.IMG_URL || 'http://localhost:9003';
+
+Router.route('/version/api')
+    .get(middleware.authenticate('user', 'guest'))
+    .get(function(req, res) {
+        var version = process.env.VERSION || 'unknown';
+        version = '{"version":"'+version+'"}'
```
Comment (melonmanchan) [inline routes/version.js]: Why not just {version: version} ? :sun_with_face:

File: routes/version.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        return res.json(200, JSON.parse(version));
+    });
+
+Router.route('/version/img')
+    .get(middleware.authenticate('user', 'guest'))
+    .get(function(req, res) {
+        var version = '{"version":"unknown"}';
```
Comment (melonmanchan) [inline routes/version.js]: Same here :new_moon_with_face:

**Comment by melonmanchan** [comment]
Goob jod

**Pull request merged**